pkg/bot/slack.go

```diff
@@ -192,7 +199,7 @@ func (b *Slack) SetNotificationsEnabled(channelName string, enabled bool) error
 
 func (sm *slackMessage) HandleMessage(b *Slack) error {
 	// Handle message only if it starts with bot mention
-	if !strings.HasPrefix(sm.Event.Text, "<@"+sm.BotID+">") {
+	if !strings.HasPrefix(sm.Event.Text, fmt.Sprintf(slackBotMentionPrefixFmt, sm.BotID)) {
```

Comment:
it doesn't change, so it would be good to move it to constructor. The best option would be to use the same thing that you have on for other bots based on my previous comment: https://github.com/kubeshop/botkube/pull/670#discussion_r937658507